docker-compose.yaml

```diff
@@ -0,0 +1,49 @@
+services:
+  api:
+    image: pramsey/pg_featureserv
```

Comment:
we should use fixed versions to ensure reproducability

Reply:
Done